lib/web_ui/lib/src/engine/scene_view.dart

```diff
@@ -308,26 +308,34 @@ final class PlatformViewContainer extends SliceContainer {
     assert(_bounds != null);
     if (_dirty) {
       final DomCSSStyleDeclaration style = container.style;
-      final double devicePixelRatio = EngineFlutterDisplay.instance.devicePixelRatio;
-      final double logicalWidth = _bounds!.width / devicePixelRatio;
-      final double logicalHeight = _bounds!.height / devicePixelRatio;
-      style.width = '${logicalWidth}px';
-      style.height = '${logicalHeight}px';
       style.position = 'absolute';
-
-      final PlatformViewPosition position = PlatformViewPosition.combine(
-        _styling!.position,
-        PlatformViewPosition.offset(_bounds!.topLeft),
-      );
+      final double devicePixelRatio = EngineFlutterDisplay.instance.devicePixelRatio;
+      final PlatformViewPosition position = _styling!.position;
 
       final ui.Offset offset = position.offset ?? ui.Offset.zero;
-      final double logicalLeft = offset.dx / devicePixelRatio;
-      final double logicalTop = offset.dy / devicePixelRatio;
-      style.left = '${logicalLeft}px';
-      style.top = '${logicalTop}px';
-
       final Matrix4? transform = position.transform;
-      style.transform = transform != null ? float64ListToCssTransform3d(transform.storage) : '';
+      if (transform == null) {
+        final ui.Offset newOffset = offset + _bounds!.topLeft;
+        final double logicalWidth = _bounds!.width / devicePixelRatio;
+        final double logicalHeight = _bounds!.height / devicePixelRatio;
+        final double logicalLeft = newOffset.dx / devicePixelRatio;
+        final double logicalTop = newOffset.dy / devicePixelRatio;
+        style.width = '${logicalWidth}px';
+        style.height = '${logicalHeight}px';
+        style.left = '${logicalLeft}px';
+        style.top = '${logicalTop}px';
+        style.transform = '';
+      } else {
+        style.width = '${_bounds!.width}px';
+        style.height = '${_bounds!.height}px';
+        style.left = '0px';
+        style.top = '0px';
```

Comment:
I think top/left are no-ops when transform is present. No need to set them (unless this is unsetting the previously set values, in which case maybe clear them so the browser doesn't need to parse anything).

Reply:
Yes, I was doing this to unset any previously set values. But if we're always using a transform in all cases as per your other suggestion, we'll never be setting these at all, so I can clean this up.